src/integTest/kotlin/io/snyk/plugin/cli/ConsoleCommandRunnerTest.kt

```diff
@@ -136,13 +137,15 @@ class ConsoleCommandRunnerTest : LightPlatformTestCase() {
         super.setUp()
         // don't report to Sentry when running this test
         unmockkAll()
+        resetSettings(project)
         mockkObject(SentryErrorReporter)
         every { SentryErrorReporter.captureException(any()) } returns SentryId()
     }
 
     @After
     override fun tearDown() {
-        super.tearDown()
         unmockkAll()
+        resetSettings(project)
+        super.tearDown()
```

Comment:
any reason to move `super.teadDown()` as last statement?

Reply:
after tearDown project will be Null. 
Not a big deal in that case but would be good to do any wrap up required project instance before that.